package.json

```diff
@@ -1,57 +1,17 @@
 {
-    "name": "unirep-social",
-    "version": "1.0.0",
-    "description": "",
-    "main": "build/core/index.js",
+    "name": "root",
```

Comment:
I think it can be named`unirep-social`

Reply:
The monorepo package shouldn't be named anything, it's not a valid package.